index.html

```diff
@@ -133,18 +135,19 @@ <h1 class="title">Projects</h1>
                 <li class="tech-btn">Ruby on Rails</li>
                 <li class="tech-btn">JavaScript</li>
             </ul>
-            <button class="see-btn" type="button">See this project</button>
+            <a class="see-btn"  href="#popup" onclick="pop()">See this project</a>
         </div>
-        <img class="project1" src="./images/Image6.svg" alt="">
+        <img class="project1" href="#popup" src="./images/Image6.svg" alt="">
         <div class="sub-div">
             <p class="subtitle">Project name goes here</p>
             <ul class="btn-list">
                 <li class="tech-btn">HTML/CSS</li>
                 <li class="tech-btn">Ruby on Rails</li>
                 <li class="tech-btn">JavaScript</li>
             </ul>
-            <button class="see-btn" type="button">See this project</button>
+            <a class="see-btn" href="#popup" onclick="pop()">See this project</a>
         </div>
+        <div class="popup" id="popup"></div>
```

Comment:
- [OPTIONAL] I think it's better to cover all your visible screens with the pop-up. Scroll in your screen just the popup if needed. The user doesn't have access to your page when the popup is shown. The only way out is to click the exit button. This will create a more professional look and a better UX. Also will remove the error when the user calls the popup several times when on the mobile version without closing the previous one. As not required by the previous reviewer I consider it optional.